geco/mips/tests/test_facility_location.py

```diff
@@ -35,3 +36,22 @@ def test_seeding(n_customers, n_facilities, ratio, seed1, seed2):
     same_seeds_produce_same_params = seed1 == seed2 and not something_different
     different_seeds_produce_different_params = seed1 != seed2 and something_different
     assert same_seeds_produce_same_params or different_seeds_produce_different_params
+
```

Comment:
OR-Lib has the optimal solutions for the files [here](http://people.brunel.ac.uk/~mastjjb/jeb/orlib/files/capopt.txt). Would be nice to check some, that can be calculated in not to much computing time.

Reply:
Please check the current code. I added a test for some instances.

Reply:
I like.